doc/python/recipe.rst

```diff
@@ -300,22 +300,22 @@ helpers in cell_parameters and make_cable_cell for building cells are used.
             # The kind method returns the type of cell with gid.
             # Note: this must agree with the type returned by cell_description.
             def cell_kind(self, gid):
-                return arbor.cell_kind.cable
+                return A.cell_kind.cable
 
             # Make a ring network
             def connections_on(self, gid):
                 src = (gid-1)%self.ncells
                 w = 0.01
                 d = 10
```

Comment:
```suggestion
                d = 10 * U.ms
```
You might also consider writing out weight, delay to make things a bit clearer to the reader.